server/audio_component.hpp

```diff
@@ -52,7 +52,7 @@ enum AUDIO_COMMANDS
 	STOP 	= 'S',
 	PAUSE 	= 'Z',
 	RESUME 	= 'R',
-
+	NEXT	= 'N',
```

Comment:
Yeah that sounds good. You can decide whether to keep it global scope, or local to a specific message/proto.